.github/workflows/_check_icons.yml

```diff
@@ -0,0 +1,34 @@
+name: Check icons
+
+on:
+  workflow_call:
+    inputs:
+      app-repository:
+        required: false
+        default: ${{ github.repository }}
+        type: string
+      repo-name:
+        required: false
+        default: ${{ github.event.repository.name }}
+        type: string
+
+jobs:
+  check_icons:
+    name: Check icons
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone workflows repository
+        uses: actions/checkout@v3
+        with:
+          repository: LedgerHQ/ledger-app-workflows
+          path: ./ledger-app-workflows
+
+      - name: Clone app repository
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.app-repository }}
```

Comment:
why not directly using ${{ github.repository }} here?

Reply:
Same for ${{ inputs.repo-name }} and in other workflows.

Reply:
It will be useful when I'll do a CI to test the test CI